Calculate_Result/Calculate_Result/AllExam.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Linq;
+
+namespace Calculate_Result
+{
+    public class SSCResult : IResultInterface
+    {
+        public double CalculateResult()
+        {
+            int[] subjectArr = new int[5];
+
+            for (int i = 0; i < subjectArr.Length; i++)
+            {
+                Console.Write($"Enter Marks Of {i + 1} Subject outof(100) => ");
+                subjectArr[i] = Convert.ToInt32(Console.ReadLine());
+            }
+            int total = subjectArr.Sum();
```

Comment:
Can we factor bellow code ?
`   int total = subjectArr.Sum();
        double percentage = total / 5;
         return percentage;`
as return subjectArr.Sum()/5; ?